.github/workflows/ci.yml

```diff
@@ -12,8 +12,11 @@ env:
   BAAS_PROJECT_ID: ${{ secrets.ATLAS_QA_PROJECT_ID}}
   REALM_CI: true
 
-jobs:
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
```

Comment:
Is this mean, that if we push to the `main` branch the old workflows on this branch will not be canceled, because the group will be by `run_id`? If yes, that's great.

Reply:
Yes - `head_ref` on `main` will be empty, so it will fall back to the `run_id`.